content/server-server-api.md

```diff
@@ -1228,7 +1228,6 @@ of `M_FORBIDDEN`.
 
 The following endpoint prefixes MUST be protected:
 
--   `/_matrix/federation/v1/send` (on a per-PDU basis)
```

Comment:
I moved this down in order to detail the behaviour of the endpoint and because the description above with HTTP 403 and `errcode` cannot actually be applied to it.